src/mscorlib/src/System/Resources/ResourceReader.cs

```diff
@@ -1373,8 +1373,9 @@ public bool MoveNext()
 
                     String key;
                     Object value = null;
+                    lock (_reader) { // locks should be taken in the same order as in RuntimeResourceSet.GetObject to avoid deadlock
                     lock (_reader._resCache) {
```

Comment:
Can you fix the indentation here. I am almost done running the tests. will merge when I get the updated PR